baseapp/server.go

```diff
@@ -69,6 +71,13 @@ func NewServer(c HTTPConfig, params ...Param) (*Server, error) {
 		base.mux.Use(middleware)
 	}
 
+	addr := c.Address + ":" + strconv.Itoa(c.Port)
+	base.server = &http.Server{
+		Addr:      addr,
+		Handler:   base.mux,
+		TLSConfig: &tls.Config{},
```

Comment:
Should I be setting this? Or should I not and leave the end consumer set it up?

Reply:
I'd leave it unset to match the default behavior we had before.